### ISSUE TITLE
Youtube sidebar covering video in Firefox

### ISSUE THREAD

**Issue by Olloboy** [xlexi/alientube#101]
![alientube](https://cloud.githubusercontent.com/assets/7129502/5885378/ce3a0b1a-a36b-11e4-921c-4e9fad4ccade.PNG)
Had this happening to me when switching from Chrome to Firefox. 
Love the plugin, just a tad hard to use with this issue.
Sorry if I have somehow posted this somewhere I shouldn't.

**Comment by Manic020** [comment]
Same thing is happening to me :/ I noticed that if the window isn't maximized, YouTube shrinks the video and it all looks normal. It's not until the window goes over a certain size that YT stretches the video and AlienTube brings the sidebar with it for whatever reason.

Screenshots: http://imgur.com/a/mwczX

**Comment by 6j76j8k5jkjk** [comment]
Yes I also have the same problem on Firefox: http://i.imgur.com/bEArAp2.png
I'm running Firefox 35

**Comment by HoWol76** [comment]
Good I'm not the only one, and also thanks for mentioning the workaround (un-maximising the browser works for me). But a proper solution would still be better.

**Comment by TedTed** [comment]
Same here. This is making alientube unusable with a tiling window manager =(

**Comment by xlexi** [comment]
This is a duplicate of #76, I am closing this ticket, refer to #76 for future updates on this issue.
